admin-dev/themes/new-theme/js/components/entity-search-input-map.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+/**
+ * These are the default selectors used as default values by the EntitySearchInput
+ * component, they are stored in an independent map file though so that they can
+ * easily be imported by other components relying on them.
+ */
+export default {
+  searchInputSelector: '.entity-search-input',
+  listSelector: '.entities-list',
+  entityItemSelector: 'li.entity-item',
+  entityDeleteSelector: '.entity-item-delete',
+};
```

Comment:
So you left the entity search input map instead of using the global one, what's the reason behind? :)

Reply:
No reason except the global one didn't exist when I started my PR ^^
I moved them in the global one now